vite.config.ts

```diff
@@ -49,6 +52,10 @@ export default defineConfig({
         }
     },
     plugins: [wasm(), solid({ ssr: false }), VitePWA(pwaOptions)],
+    define: {
+        "import.meta.env.__COMMIT_HASH__": JSON.stringify(commitHash),
+        "import.meta.env.__RELEASE_VERSION__": JSON.stringify("0.4.16")
```

Comment:
looks like there's a pretty nice way to get this direct from package.json. one less thing to update every version bump! https://stackoverflow.com/questions/67194082/how-can-i-display-the-current-app-version-from-package-json-to-the-user-using-vi

Reply:
well thats super easy